keras/callbacks.py

```diff
@@ -1232,3 +1235,72 @@ def __init__(self,
             self.on_train_end = on_train_end
         else:
             self.on_train_end = lambda logs: None
+
+
+class MXNetModelCheckpoint(ModelCheckpoint):
+    """Save the MXNet Model after every epoch. Saves both params and symbol file. Saves in the current directory.
+
+       If save_best_only is True, saves the MXNet model in the format '<prefix>-symbol.json' and '<prefix>-0000.params'.
+       i.e., uses 0000 as placeholder for the epoch. You will have only one best model at the end of training.
+
+       If save_best_only is False, i.e., you want to save the MXNet model after each epoch, this callback saves the Model
+       in the format '<prefix>-symbol.json' and '<prefix>-<epoch>.params'. Example: If you are running the training job
+       for 2 epochs, you will have one symbol file - '<prefix>-symbol.json' and 2 params file, one each for the 2 epochs,
+       '<prefix>-0001.params' and '<prefix>-0002.params'
```

Comment:
looks like we are starting with epoch 0 - so files saved are <prefix>-0000.params & -0001.params - please update accordingly

Reply:
Thanks. Fixed.